test/integration/dispatcher_integration_test.cc

```diff
@@ -229,5 +229,9 @@ TEST_P(DispatcherIntegrationTest, RaceDoesNotCauseDoubleDeletion) {
   http_dispatcher_.synchronizer().signal("dispatch_encode_final_data");
 }
 
+// TODO(junr03): test with envoy local reply with local stream not close, which causes a reset fired
```

Comment:
```suggestion
// TODO(junr03): test with envoy local reply with local stream not closed, which causes a reset fired
```

Reply:
if you are ok with it, I will fix this in #791 to avoid re-running CI